salt/fileclient.py

```diff
@@ -592,20 +592,37 @@ def swift_opt(key, default):
         destfp = None
         try:
             # Tornado calls streaming_callback on redirect response bodies.
-            # But we need streaming to support fetching large files (> RAM avail).
-            # Here we working this around by disabling recording the body for redirections.
-            # The issue is fixed in Tornado 4.3.0 so on_header callback could be removed
-            # when we'll deprecate Tornado<4.3.0.
-            # See #27093 and #30431 for details.
-
-            # Use list here to make it writable inside the on_header callback. Simple bool doesn't
-            # work here: on_header creates a new local variable instead. This could be avoided in
-            # Py3 with 'nonlocal' statement. There is no Py2 alternative for this.
+            # But we need streaming to support fetching large files (> RAM
+            # avail).  Here we working this around by disabling recording the
```

Comment:
Noted. This is only really showing up in the diff because I balanced the line endings when I added all the extra documentation. When I was troubleshooting this, I wanted to make our approach easier to understand, because it wasn't really documented at all.

Just pushed a commit that fixes this.